src/middlewares/socialCallback.js

```diff
@@ -0,0 +1,27 @@
+import bcrypt from 'bcryptjs';
+import db from '../models';
+
+const { User } = db;
+
+
+const strategyCallback = async (accessToken, refreshToken, profile, done) => {
+  const {
+    id, displayName, photos, emails
+  } = profile;
+
+  const salt = bcrypt.genSaltSync(10);
+  const hashPassword = bcrypt.hashSync(id, salt);
+  const [user] = await User.findOrCreate({
```

Comment:
@ssanusi Nice work. Have you considered destructuring to make the code more readable

Reply:
the above user is destructured